engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -63,6 +63,11 @@ public boolean compare(StateChange stateChange) {
     }
 
     @Override
+    public boolean isTheDefaultInstance() {
+        return this == defaultInstance;
+    }
```

Comment:
Ok, getDefaultInstance() and isTheDefaultInstance() could go into an abstract class now. =)

But don't bother about it right now. You might want to do it when you start implementing all the other state changes, to save some typing/cut&pasting.